### PR TITLE
IMF SDR Source

### PR DESCRIPTION
**Pull request by zmanian**
Builds off #5 

Provides a source for International Monetary Fund's Special Drawing Rights

Uses `rust-csv` to parse the tab seperated value spreadsheet formatted data.

### CODE REVIEW

File: src/sources/imf_sdr.rs [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    /// `trading_pairs()` test with known currency pair
+    #[test]
+    #[ignore]
+    fn trading_pairs_ok() {
+        let pair = "KRW/SDR".parse().unwrap();
+        let quote = block_on(ImfSDRSource::new().trading_pairs(&pair)).unwrap();
+        dbg!(&quote);
+    }
+
+    // `trading_pairs()` with invalid currency pair
```
Comment (tony-iqlusion) [inline src/sources/imf_sdr.rs]: Some issues getting this working?

Comment (zmanian) [reply src/sources/imf_sdr.rs]: I'm not sure if this test really makes sense. 

We are just grabbing a tsv file from a server.

If you query for an invalid currency, it just won't be found by the parser.

Comment (tony-iqlusion) [reply src/sources/imf_sdr.rs]: The main reason I think it'd be nice to have is ensuring that 404 response handling doesn't panic

**Comment by zmanian** [comment]
Closed in favor of #53

**Pull request closed without merging**